dotCMS/src/main/java/com/dotcms/rest/ESIndexResource.java

```diff
@@ -94,8 +96,6 @@ public void run() {
                             HibernateUtil.closeSession();
```

Comment:
shouldn't be removed this one HibernateUtil.closeSession(); too?

Reply:
@jdotcms @acurionedotcms two things: 
1. Entire `finally` block needs to me removed.
2. The annotation should be at run-method level, since this is a separate thread. Putting it at restoreIndex-method level won't take effect